proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,219 @@
+# AWS CLI Contributing Guide
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kenneth Daily
+**Status**       | Proposed
+**Created**      | 19-October-2021
+
+## Abstract
+
+This document proposes standards and processes to improve and publish community
+expectations for contributions. It defines the process for proposing,
+submitting, and reviewing a contribution. This proposal will make it easier for
+the community to successfully contribute to the AWS CLI.
+
+## Motivation
+
+### Goals
+
+The approach for customer contributions should satisfy the following goals:
+
+1. Users should know how they can help. Contributions come in different shapes
+   and sizes and are not limited to code. 
+2. It should be clear where there are opportunities to provide feedback. This
+   includes well structured comments, use cases, and voting.
+3. We should be fixing and improving the things that people want based on
+   feedback they provide via comments and votes on issues.
+4. It should be easy for non-maintainers to make high quality contributions.
+   Contributors come with different levels of experience and from different
+   backgrounds, so we should lower the barrier for participation and the process
+   towards acceptance.
+5. Users should know exactly what to expect when interacting with the
+   maintainers. The next steps should be clearly defined.
+6. Communication from maintainers should happen in a timely manner, even
+   (especially!) if that’s saying no. An initial response to open PRs should
+   occur within [TBD] business days.
+7. Questions and friction should be reduced with automation wherever possible
+   and documented when not.
+
+### Non-goals
+
+### Motivating Examples
+
+#### Limited information on where to contribute
+
+The AWS CLI currently has code, documentation, and examples published publicly
+on a number of GitHub repositories. We have hundreds of open issues and pull
+requests across these areas. However, we do not have a guide for users to know
```

Comment:
It would be good to enumerate specific examples of what we would not accept to help illustrate this.